docs/index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset='utf-8'>
+    <meta http-equiv="X-UA-Compatible" content="chrome=1">
+
+    <link rel="stylesheet" type="text/css" href="stylesheets/stylesheet.css" media="screen">
+    <link rel="stylesheet" type="text/css" href="stylesheets/github-dark.css" media="screen">
+    <link rel="stylesheet" type="text/css" href="stylesheets/print.css" media="print">
+
+    <title>Refocus by salesforce</title>
+  </head>
+
+  <body>
+
+    <header>
+      <div class="container">
+        <h1>Refocus</h1>
+        <h2>The Go-To Platform for Visualizing Service Health</h2>
+
+        <section id="downloads">
+          <a href="https://github.com/salesforce/refocus/zipball/master" class="btn">Download as .zip</a>
+          <a href="https://github.com/salesforce/refocus/tarball/master" class="btn">Download as .tar.gz</a>
+          <a href="https://github.com/salesforce/refocus" class="btn btn-github"><span class="icon"></span>View on GitHub</a>
+        </section>
+      </div>
+    </header>
+
+    <div class="container">
```

Comment:
You might want to delete the default boilerplate here, especially now that this is public and showing up at http://salesforce.github.io/refocus :)